### DISCUSSION TITLE
可以 | /cn/2022/01/affirmation/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1456] [Comments]
我问三岁的阿懵
今天吃白米饭可以吗
我以为照旧会是那句
我听了两年的大声坚决的 No
然而　只听见他柔柔地回答
可以
刹那间
我就像听见了婚礼上的那句
我愿意

https://yihui.org/cn/2022/01/affirmation/

**Comment by liechi** [comment]
前几天和我的 github 关联的邮箱收到几千条“骚扰”信息。掐指一算，估计是你在给网站动手术。今天一看，果然。

于是来试试看好使不。

嗯，可以。

**Reply by yihui** [reply]
是的，这几天刚把一万多条 Disqus 评论迁移到了 Github，并启用了 Giscus。接下来我会写篇日志说明一下。打扰啦！

**Comment by LFG12** [comment]
哇，说起来这种感觉，有幸也感受到过呀。
